### DISCUSSION TITLE
Removing "m3u -" prefix from playlist names

### DISCUSSION THREAD

**Discussion by JOJ0** [ampache/ampache#3872] [Q&A] [answered]
is there a configuration option to "not add" the "m3u -" prefix when importing playlists upon catalog "add"?

![Screenshot 2024-03-12 at 13 28 44](https://github.com/ampache/ampache/assets/2733783/e37bf91c-86c9-438c-b9a0-5bec82588e49)

(Unfortunately I'm on a quite old Ampache version 4.x but answers regarding any version are welcome, I might be able to backport/quickfix the feature)

**Comment by lachlan-00** [comment, marked as answer]
the current code is to import using filename as the playlist name.

in the older code it's in the catalog class
![image](https://github.com/ampache/ampache/assets/1305249/6653d6ae-5063-43a6-902a-3e083f279164)
